strider/server.py

```diff
@@ -348,7 +350,12 @@ async def async_lookup(
     redis_client: Redis,
 ):
     """Perform lookup and send results to callback url"""
-    query_results = await lookup(query_dict, redis_client)
+    try:
+        query_results = await asyncio.wait_for(
+            lookup(query_dict, redis_client), timeout=max_process_time
+        )
+    except asyncio.TimeoutError:
```

Comment:
Would this fail on line 353 because `query_results` is undefined? We should send some sort of timeout message back to aragorn. Same with the other functions

Reply:
I'm not sure what the timeout message should be. We do have usage of the "strider_commuincation" field, so we could add something using that.